app/knowledge_panels.py

```diff
@@ -118,13 +136,22 @@ def last_edits_kp(
     if value is not None:
         query[f"{facet_plural(facet=facet)}_tags_en"] = value
         description += f" {value}"
+        source_url = f"{url}/{facet}/{value}?sort_by=last_modified_t"
+    if sec_facet is not None:
+        description += f" {sec_facet}"
```

Comment:
add it to description only  if sec_value is not None (because otherwise it does not mean much, as it has no impact on search)